clang-tools-extra/docs/clang-tidy/checks/cppcoreguidelines/pro-type-static-cast-downcast.rst

```diff
@@ -14,3 +14,11 @@ unrelated type ``Z``.
 This rule is part of the `Type safety (Type.2)
 <https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Pro-type-downcast>`_
 profile from the C++ Core Guidelines.
+
+Options
+-------
+
+.. option:: StrictMode
+
+  When set to `false`, no warnings are emitted for casts on non-polymorphic
+  types. Default is `true`.
```

Comment:
Why "Default is `true`"? I think Default false will keep the same behavior as old version.

Reply:
StrictMode=true, result in more warnings being produced (this is current behavior).
StickMode=false, will cause casts on non-polymorphic types to be ignored.